src/components/Cards.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <div id="backlog-container" style="overflow: scroll;">
+        <div v-for="el in onCategory(category, cards)" :key="el.id" class="card-content border-radius-card h-25 w-90 card">
```

Comment:
Komponen ini juga kurang tepat potong2 komponenya, yang div lo v-for ini mungkin bisa lo namakan TaskCard, dan TaskCard ini nge loopingnya di dalam komponen CardContainer yang gw rekomendasi di komen sebelumnya